common/src/pallet_tests.rs

```diff
@@ -174,3 +174,127 @@ macro_rules! impl_pallet_authorship_inner {
         $crate::impl_pallet_authorship_inner!($runtime, $($( $rest )*)?);
     };
 }
+
+#[macro_export]
+macro_rules! impl_pallet_staking {
+    ($( $tokens:tt )*) => {
+        #[allow(dead_code)]
+        pub struct DummyEraPayout;
+        impl pallet_staking::EraPayout<u128> for DummyEraPayout {
+            fn era_payout(
+                _total_staked: u128,
+                total_issuance: u128,
+                _era_duration_millis: u64,
+            ) -> (u128, u128) {
+                // At each era have 1% `total_issuance` increase
+                (Permill::from_percent(1) * total_issuance, 0)
+            }
+        }
+
+        type X = (AccountId, BlockNumber, pallet_staking::Pallet<Test>, ConstU32<100>);
```

Comment:
I understand but perhaps it would be better to give it a longer name

Reply:
Sure, it came across my mind as well before I forgot about it ))